src/templates/dso_api/dynamic_api/mvt_index.html

```diff
@@ -9,7 +9,7 @@
     <h1>Index of datasets supporting Mapbox Vector Tiles (MVT)</h1>
 
     {% for name, fields, schema in datasets %}
-      <h2>{% if schema.title %}{{ schema.title }}{% else %}{{ name|title }}{% endif %}</h2>
+      <h2><a href="{{ request.scheme }}://{{ request.get_host }}{{ request.path }}{{ name }}">{% if schema.title %}{{ schema.title }}{% else %}{{ name|title }}{% endif %}</a></h2>
```

Comment:
```suggestion
      <h2><a href="{% url 'dynamic_api:mvt-single-dataset' dataset_name=name %}">{% if schema.title %}{{ schema.title }}{% else %}{{ name|title }}{% endif %}</a></h2>
```

Als dat mogelijk is. Anders mag `{{ request.scheme }}://{{ request.get_host }}` sowieso weg. :-)